rust_src/src/xfaces.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{
```

Comment:
1) The file should begin with `//! Description`. Grab the one from the top of the C file and update as appropriate.
2) Our style is to list std, then external crates, then our crates, then the src/ modules.

Reply:
https://github.com/r-darwish/remacs/blob/clear-face-cache/src/xfaces.c#L22

Should I just write the first line, which is "Faces.", or should I convert the whole paragraph?

Reply:
`"Face" primitives.` is what you want.